packages/serverless-contracts/src/contracts/apiGateway/__tests__/httpLambdaHandler.test.ts

```diff
@@ -41,7 +41,7 @@ describe('apiGateway lambda handler', () => {
           requestContext,
         }) => {
           await Promise.resolve();
-          const myCustomClaim = requestContext.authorizer.claims.foo ?? '';
+          const myCustomClaim = requestContext.authorizer.claims.foo;
```

Comment:
Could you test with another property? This kind of invalidates the former test, which checks that we get the correct type depending on the authorizer